src/templates/template-common/README.md

```diff
@@ -68,11 +68,16 @@ torchrun \
   --node_rank 0 \
   --master_addr #:::= it.master_addr :::# \
   --master_port #:::= it.master_port :::# \
-  main.py config.yaml --backend #:::= it.backend :::# \
   #::: if ((it.argparser == 'fire')) { :::#
+  main.py config.yaml --backend #:::= it.backend :::# \
   [--override_arg=value]
-
+  #::: } else if ((it.argparser == 'hydra')){ :::#
+  main.py  --config-dir=[dir-path] \
+  --config-name=[config-name] ++backend= #:::= it.backend :::# ++override_arg=[value]
```

Comment:
can we use `--backend` to specify override arg ?

Reply:
No as Hydra uses `++` to override the config as mentioned here https://hydra.cc/docs/tutorials/basic/your_first_app/config_file/